lisa/features/infiniband.py

```diff
@@ -324,10 +329,12 @@ def _install_dependencies(self) -> None:
                 if node.os.is_package_in_repo(package):
                     ubuntu_required_packages.append(package)
             node.os.install_packages(ubuntu_required_packages)
+        elif isinstance(node.os, CBLMariner):
+            node.os.install_packages(cblmariner_required_packages)
         else:
             raise UnsupportedDistroException(
                 node.os,
-                "Only CentOS 7.6-8.3 and Ubuntu 18.04-20.04 distros are "
+                "Only CentOS 7.6-8.3, Ubuntu 18.04-22.04 and CBLMariner distros are "
                 "supported by the HCP team",
```

Comment:
Fix the typo "HCP" to "HPC".

Reply:
I fixed the typo and refined the error log since CBLMariner is not supported by the HPC team yet. There's no CBLMariner HPC image.